proxy/src/main/java/com/wavefront/agent/preprocessor/SpanRenameTagTransformer.java

```diff
@@ -0,0 +1,67 @@
+package com.wavefront.agent.preprocessor;
+
+import com.google.common.base.Function;
+import com.google.common.base.Preconditions;
+
+import com.yammer.metrics.core.Counter;
+
+import java.util.regex.Pattern;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import wavefront.report.Span;
+
+/**
+ * Rename a given span tag's/annotation's (optional: if its value matches a regex pattern)
+ *
+ * If the tag matches multiple span annotation keys , all keys will be renamed.
+ *
+ * @author akodali@vmare.com
+ */
+public class SpanRenameTagTransformer implements Function<Span, Span> {
```

Comment:
Let's name it SpanRenameAnnotationTransformer so it's consistent with the rest of span rules